### ISSUE TITLE
Search results repeat, no easy way to delete 

### ISSUE THREAD

**Issue by maxliqd** [liqd/digitalstrategie#714]
**Essential Information** Search results currently show all hits, not the pages, this leads to a lot of repeated pages in the results.  You can also not delete what you entered with an "x" in the search bar. 
**URL:** gemeinsamdigital.berlin.de
**device & browser:**
**operating system (incl. version):**
**behaviour:** all hits are individual results, no way to easily delete the search
**expected behaviour:**
- Show pages only once
- add x to delete what was entered
- Nice to have: add number of results
- Nice to have: adjust wording to "Suche nach..." / "Search for..."

**Additional information if relevant**
**user role:**
**important screensize:**
**Comment/Question:**

**Screenshot if design related**:
Since issues are public, make sure to not display personal or secret data.
